modules/kafka/src/main/java/com/devonfw/module/producer/ProducerPayload.java

```diff
@@ -0,0 +1,136 @@
+package com.devonfw.module.producer;
+
+/**
+ * @author ravicm
+ *
+ */
+public class ProducerPayload {
+
+  private Object message;
+
+  /**
+   * Topic name. If topic name is not provided then default name will be assigned.
+   */
+  private String topic;
```

Comment:
Nice that in this case you actually add JavaDoc that is helpful. However, this should go to the getter and not stay here where it never gets into the generated JavaDocs and nobody can ever see it in IDE auto-completion.